console-plugins/vi-map-visualization-plugin/include/vi-map-visualization-plugin/visualization-plugin.h

```diff
@@ -31,6 +30,8 @@ class VisualizationPlugin : public common::ConsolePluginBase {
   int visualizeSensorExtrinsics() const;
   int visualizeLandmarkObserverRays() const;
   int visualizeReprojectedDepthResource(backend::ResourceType type);
+  int visualizeReprojectedDepthResourceFromMission(backend::ResourceType type);
+  int visualizeReprojectedDepthResourceSequentially(backend::ResourceType type);
```

Comment:
Maybe I'm missing something obvious but I don't see where these functions are defined?

Reply:
good catch! I forgot to copy something, my bad!